README.md

```diff
@@ -29,6 +29,98 @@ Official documentation on how to use this provider can be found on the
 [Terraform
 Registry](https://registry.terraform.io/providers/argoproj-labs/argocd/latest/docs).
 
+## Upgrading
+
+### Migrate provider source `oboukili` -> `argoproj-labs`
+
+As announced in the releases [v6.2.0] and [v7.0.0], we moved the provider from "github.com/**oboukili**/terraform-provider-argocd/" 
+to "github.com/**argoproj-labs**/terraform-provider-argocd". Users need to migrate their Terraform state according to
+HashiCorps [replace-provider] docs. In summary, you can do the following:
+
+1. List currently used providers
+
+    ```bash
+    $ terraform providers
+
+    Providers required by configuration:
+    .
+    ├── provider[registry.terraform.io/hashicorp/helm] 2.15.0
+    ├── (..)
+    └── provider[registry.terraform.io/oboukili/argocd] 6.1.1
+
+    Providers required by state:
+
+        (..)
+
+        provider[registry.terraform.io/oboukili/argocd]
+
+        provider[registry.terraform.io/hashicorp/helm]
+    ```
+
+2. **If you see** the provider "registry.terraform.io/**oboukili**/argocd", you can update the provider specification:
+
+    ```diff
+    --- a/versions.tf
+    +++ b/versions.tf
+    @@ -5,7 +5,7 @@ terraform {
+        }
+        argocd = {
+    -      source  = "oboukili/argocd"
+    +      source  = "argoproj-labs/argocd"
+        version = "6.1.1"
```

Comment:
This will not result in the following log lines, 
```suggestion
        version = "~> 7"
```

Will only download the argoproj version (needs testing)

Reply:
I think you need both providers available during the `terraform state replace-provider` execution. But I can test that

Reply:
It does not work:
```bash
$ terraform init
Initializing HCP Terraform...
Initializing provider plugins...
- Finding hashicorp/tls versions matching "4.0.6"...
- Finding akeyless-community/akeyless versions matching "1.7.4"...
- Finding argoproj-labs/argocd versions matching "~> 7.0"...
- Finding hashicorp/helm versions matching "2.15.0"...
- Finding latest version of hashicorp/time...
- Finding latest version of oboukili/argocd...
- Finding hashicorp/kubernetes versions matching "2.32.0"...
- Finding hashicorp/null versions matching "3.2.3"...
- Installing hashicorp/tls v4.0.6...
- Installed hashicorp/tls v4.0.6 (signed by HashiCorp)
- Installing akeyless-community/akeyless v1.7.4...
- Installed akeyless-community/akeyless v1.7.4 (self-signed, key ID EEA698E7ACFEAB58)
- Installing argoproj-labs/argocd v7.0.0...
- Installed argoproj-labs/argocd v7.0.0 (self-signed, key ID 6421DA8DFD8F48D0)
- Installing hashicorp/helm v2.15.0...
- Installed hashicorp/helm v2.15.0 (signed by HashiCorp)
- Installing hashicorp/time v0.12.1...
- Installed hashicorp/time v0.12.1 (signed by HashiCorp)
- Installing hashicorp/kubernetes v2.32.0...
- Installed hashicorp/kubernetes v2.32.0 (signed by HashiCorp)
- Installing hashicorp/null v3.2.3...
- Installed hashicorp/null v3.2.3 (signed by HashiCorp)
Partner and community providers are signed by their developers.
If you'd like to know more about provider signing, you can read about it here:
https://www.terraform.io/docs/cli/plugins/signing.html
╷
│ Error: Incompatible provider version
│
│ Provider registry.terraform.io/oboukili/argocd v7.0.0 does not have a package available for your current platform, darwin_amd64.
│
│ Provider releases are separate from Terraform CLI releases, so not all providers are available for all platforms. Other versions of this provider may have different platforms supported.
╵
```